### PR TITLE
Nightly 11 1 20

### PR DESCRIPTION
**Pull request by Eutherin**
added a bunch of block protections, added nether warts to farmable blocks

### CODE REVIEW

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/192

**Comment by Eutherin** [comment]
https://github.com/Adriftus-Studios/network-script-data/issues/190

File: denizen_scripts/survival/repo_link/items/backpack.dsc [hunk 1]
```diff
@@ -91,6 +91,8 @@ Backpack_events:
         - determine cancelled
     on player drops backpack_*:
     - determine cancelled
+    on player prepares anvil craft backpack_*:
+    - determine <element[9001]>
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo_link/items/backpack.dsc:95]: you don't need the wrapper here
```suggestion
    - determine 9001
```

File: denizen_scripts/survival/repo_link/global_systems/chunk_claim_protection.dsc [hunk 2]
```diff
@@ -1200,7 +1200,7 @@ claiming_protection_events:
     - if !<yaml[claims].read[groups.<[group]>.members.<player.uuid>.break]||false> && !<yaml[claims].read[groups.<[group]>.members.everyone.break]>:
       - narrate "<&c>You do not have permission to trample crops here."
       - determine cancelled
-    on player clicks *door|*_button|lever|chest|ender_chest|*_gate|crafting_table|anvil|furnace|brewing_stand|enchanting_table|*_bed|bookshelf|barrel|hopper priority:100:
+    on player clicks *door|*_button|lever|chest|ender_chest|*_gate|crafting_table|anvil|furnace|brewing_stand|enchanting_table|*_bed|bookshelf|barrel|hopper|tnt|dispenser|dropper|stonecutter|lodeston|beehive|smithing_table|lectern|grindstone|flethcing_table|cartography_table|blast_furnace|smoker|composter|loom|trappged_chest|*_shulker_box|cauldron|cake|bell|jukebox|note_block|beacon|respawn-anchor|repeater|redstone_dust|comparator|daylight_detector|tripwire_hook|end_portal_frame|flower_pot|bee_nest|armor_stand|item_frame|*minecart|*boat| priority:100:
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo_link/global_systems/chunk_claim_protection.dsc:1203]: there's a stray pipe at the end here that would mess this up here
```suggestion
    on player clicks *door|*_button|lever|chest|ender_chest|*_gate|crafting_table|anvil|furnace|brewing_stand|enchanting_table|*_bed|bookshelf|barrel|hopper|tnt|dispenser|dropper|stonecutter|lodeston|beehive|smithing_table|lectern|grindstone|flethcing_table|cartography_table|blast_furnace|smoker|composter|loom|trappged_chest|*_shulker_box|cauldron|cake|bell|jukebox|note_block|beacon|respawn-anchor|repeater|redstone_dust|comparator|daylight_detector|tripwire_hook|end_portal_frame|flower_pot|bee_nest|armor_stand|item_frame|*minecart|*boat priority:100:
```

**Review by BehrRiley — CHANGES_REQUESTED** [review]

**Review by BehrRiley — APPROVED** [review]

**Pull request merged**